### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by wxk-308** [chenshuo/muduo#695] [General]
Buffer.h
void makeSpace(size_t len）函数中下面语句是不是应该调换顺序赋值？
不然writeIndex_不能够指向正确位置
writerIndex_ = kCheapPrepend + readabel;
readerIndex_ = kCheapPrepend;

**Comment by chenshuo** [comment]
你能否提供一个实际案例，证明现在的代码有 bug ？

**Reply by wxk-308** [reply]
可能是我没能理解您的设计思路

**Reply by chenshuo** [reply]
1. 如果你认为现在的代码有 bug，你可以构造一个例子 (test case)，来暴露这个 “writeIndex_不能够指向正确位置” 的 bug。

2. 如果你认为现在的代码没问题，可以让管理员 close issue。

请问，你现在持有的观点是 1 还是 2 ？

**Reply by wxk-308** [reply]
我理解错了，您的代码没有问题

**Comment by wxk-308** [comment]
观点二发自我的 iPhone在 2023年9月21日，01:09，Shuo Chen ***@***.***> 写道：﻿


如果你认为现在的代码有 bug，你可以构造一个例子 (test case)，来暴露这个 “writeIndex_不能够指向正确位置” 的 bug。


如果你认为现在的代码没问题，可以让管理员 close issue。


请问，你现在持有的观点是 1 还是 2 ？
